src/btree/bt_delete.c

```diff
@@ -316,8 +315,8 @@ __wt_delete_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
         __wt_page_modify_set(session, page);
 
     if (ref->page_del != NULL && ref->page_del->prepare_state != WT_PREPARE_INIT) {
-        WT_STAT_CONN_INCR(session, cache_read_deleted_prepared);
-        WT_STAT_DATA_INCR(session, cache_read_deleted_prepared);
+
```

Comment:
Nit: There's some redundant whitespace added in this commit. Can we go through and remove them.